### ISSUE TITLE
add new feature for tfdbg that ignoring specific layers

### ISSUE THREAD

**Issue by mercy0387** [tensorflow/tensorflow#16619]
Using tfdbg, `has_inf_or_nan` filter is very useful. But when I need `inf` at some calculations in the network explicitly, this filter catches `inf` every time. (e.g. https://github.com/keras-team/keras/issues/9161 )

So I want to suggest new feature, `ignore option`, to `add_tensor_filter`. (I think it's around [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/debug/wrappers/local_cli_wrapper.py#L193). right?)

This option can take layer's names and layers which is selected to this option are ignored explicitly when tfdbg check `inf_or_nan` in the network.

This is also useful other than in my case above. If my understanding is not correct, let me know.

Thanks.

**Comment by caisq** [comment]
@mercy0387 thanks for filing the feature request. I agree that adding arguments to tfdbg filters like `has_inf_or_nan` will be useful. I will work on the new feature.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
